test/acceptance/features/steps/steps.py

```diff
@@ -466,3 +468,12 @@ def validate_secret_empty(context, secret_name):
                       ignore_exceptions=(json.JSONDecodeError,))
     except polling2.TimeoutException:
         pass
+
+
+@given(u'"{app_name}" is deployed from image "{image_ref}"')
+def create_deployment(context, app_name, image_ref):
+    app = App(app_name, context.namespace.name, image_ref)
+    if not app.is_running:
+        assert False, "Deployment is already present."
+    else:
+        assert app.install() is True, "Failed to create deployment."
```

Comment:
We should not fail if app is not running, instead we should try to install it and then check if running, similar to https://github.com/redhat-developer/service-binding-operator/blob/f1c3975476de43b2e305ee03c29632065291392c/test/acceptance/features/steps/generic_testapp.py#L24

Reply:
Updated.